content/en/docs/contribute/style/content-guide.md

```diff
@@ -10,107 +10,67 @@ card:
-        - Linking to inactive projects in the kubernetes and kubernetes-sigs GitHub organizations
-        - Linking to open source projects that are not CNCF projects or do not reside in the kubernetes or kubernetes-sigs GitHub organizations
-1. Content about training courses
-    - Allowed:
-        - Linking to vendor-neutral Kubernetes training courses offered by the [CNCF](https://www.cncf.io/), the [Linux Foundation](https://www.linuxfoundation.org/), and the [Linux Academy](https://linuxacademy.com/), which is a partner of the Linux Foundation
-            - Example: linking to Linux Academy courses such as [Kubernetes Quick Start](https://linuxacademy.com/course/kubernetes-quick-start/) and [Kubernetes Security](https://linuxacademy.com/course/kubernetes-security/)
-    - Not Allowed:
-        - Linking to online training outside of the CNCF, the Linux Foundation, or the Linux Academy; the Kubernetes documentation does not link to third-party content
-            - Example: linking to Kubernetes tutorials or courses on Medium, KodeKloud, Udacity, Coursera, learnk8s, and similar websites
-        - Linking to vendor-specific tutorials regardless of the training provider
-            - Example: linking to Linux Academy courses such as [Google Kubernetes Engine Deep Dive](https://linuxacademy.com/google-cloud-platform/training/course/name/google-kubernetes-engine-deep-dive) and [Amazon EKS Deep Dive](https://linuxacademy.com/course/amazon-eks-deep-dive/)
+majority of Kubernetes documentation is specific to the [Kubernetes
+project](https://github.com/kubernetes/kubernetes). 
+
+## What's allowed
+
+Kubernetes docs permit only some kinds of content.
+
+### Third party content
+
+Kubernetes documentation includes applied examples of projects in the Kubernetes project&emdash;projects that live in the [kubernetes](https://github.com/kubernetes) and
+[kubernetes-sigs](https://github.com/kubernetes-sigs) GitHub organizations. 
+
+Links to active content in the Kubernetes project are always allowed. 
+
+Kubernetes requires some third party content to function. Examples include Docker, 
+[networking policy](/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/) (CNI plugins), [Ingress controllers](https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/), and [logging](https://kubernetes.io/docs/concepts/cluster-administration/logging/).
+
+Docs can link to third-party content outside the Kubernetes project if it's necessary for Kubernetes to function in-project.
+
+### Dual sourced content
+
+Wherever possible, Kubernetes docs link to canonical sources instead of hosting
+dual-sourced content.
+
+Dual-sourced content requires double the effort (or more!) to maintain
+and grows stale more quickly.
+
+{{< note >}}
+
+If you're a maintainer for a Kubernetes project and need help hosting your own docs, 
+ask for help in [SIG Docs Slack](https://app.slack.com/client/T09NY5SBT/C1J0BPD2M). 
```

Comment:
```suggestion
ask for help in ['#sig-docs on Kubernetes Slack](https://kubernetes.slack.com/messages/C1J0BPD2M/).
```
Maybe bring forward the link to http://slack.k8s.io/ as well?

Reply:
@sftim 
> if you like, I can propose more changes to this PR as commits you can cherry pick?

Suggestions work!

> does it make sense to open a new branch and merge that in to master once it's ready?

I'm not sure I follow. Can you restate?

Reply:
I'm wondering if we want to land a lot of KEP-1327 changes at once. If that's the outcome we intend, I think the work would look like:
1. create branch
2. create several PRs that target that branch, each implementing a piece of KEP 1327
3. merge branch to _master_

I think it makes sense to implement, if you like, a minimum viable excision, and follow that up with more PRs to make the website match the revised content guide. Is that “minimum viable excision” what you'd intended this PR to become?

Reply:
Also, GitHub suggestions are limited in what they let you do and are vulnerable to pink unicorn syndrome (GitHub server-side failure).

Reply:
@sftim +1 to the umbrella branch approach. The branch will need to run long, essentially the duration of 1.19 in order to give content owners time to revise or re-home their content. We'll have to maintain it like a release branch (i.e., merge master in periodically). 

> I think it makes sense to implement, if you like, a minimum viable excision, and follow that up with more PRs to make the website match the revised content guide. Is that “minimum viable excision” what you'd intended this PR to become?

I think an MVE is a good idea, though it's likely to be quite minimal indeed.